#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_mib_tip.dpatch by Balint Reczey <balint@balintreczey.hu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Suggest solutions on the missing MIBs popup window.

@DPATCH@
--- ./epan/oids.c	(revision 31246)
+++ ./epan/oids.c	(working copy)
@@ -571,7 +571,10 @@
 
 	if (smi_errors->len) {
 		report_failure("The following errors were found while loading the MIBS:\n%s\n\n"
-					   "The Current Path is: %s\n" , smi_errors->str , path_str);
+					   "The Current Path is: %s\n\nYou can avoid this error message "
+					   "by removing the missing MIB modules at Edit -> Preferences"
+					   " -> Name Resolution -> SMI (MIB and PIB) modules or by "
+					   "installing them using the snmp-mibs-downloader package.\n" , smi_errors->str , path_str);
 		D(1,("Errors while loading:\n%s\n",smi_errors->str));
 	}
 
